utils/service/txmgr/cli.go

```diff
@@ -34,21 +34,6 @@ const (
 	ReceiptQueryIntervalFlagName      = "txmgr.receipt-query-interval"
 )
 
-var (
-	BatcherHDPathFlag = cli.StringFlag{
-		Name: "batcher-hd-path",
-		Usage: "DEPRECATED: The HD path used to derive the batcher wallet from the " +
-			"mnemonic. The mnemonic flag must also be set.",
-		EnvVar: "BATCHER_HD_PATH",
-	}
-	ValidatorHDPathFlag = cli.StringFlag{
-		Name: "validator-hd-path",
-		Usage: "DEPRECATED: The HD path used to derive the validator wallet from the " +
-			"mnemonic. The mnemonic flag must also be set.",
-		EnvVar: "VALIDATOR_HD_PATH",
-	}
-)
```

Comment:
I removed some legacy flags like `BatcherHDPathFlag`, `ValidatorHDPathFlag` cause it is duplicated with `HDPath`.